.github/workflows/lint-ts.yml

```diff
@@ -34,7 +34,7 @@ jobs:
 
             - name: Install dependencies
               run: |
-                  npm install @typescript-eslint/parser @typescript-eslint/eslint-plugin eslint-plugin-tsdoc eslint typescript eslint-plugin-import@latest eslint-config-prettier prettier
+                  npm install eslint-plugin-import @eslint/compat globals @eslint/js @eslint/eslintrc @typescript-eslint/parser @typescript-eslint/eslint-plugin eslint-plugin-tsdoc eslint typescript eslint-config-prettier prettier
```

Comment:
Q: why do we need to provide this list? isn't it enough to run npm install in the node package?

Reply:
Its not in the node package, we use npx so i need it in source, instead of for each folder.
We can add package.json to the root folder, but i have mixed feeling about it since its not a npm project.

Reply:
I agree that this ugly and when ill have more time ill do some cleaning, not sure that all are still needed